docs/asciidoc/front_matter.adoc

```diff
@@ -11,16 +20,100 @@ The primary functionality of DRS is to map a logical ID to a means for physicall
+
+2) The client makes API calls to a meta-resolver to look up the URL pattern for the namespace. (See <<_appendix_compact_identifier_based_uris>> for details.) The URL pattern is a string containing a `{$id}` parameter, such as:
+
+    https://drs.myexample.org/ga4gh/drs/v1/objects/{$id}
+
+3) The client generates a DRS URL from the URL template by replacing {$id} with the accession it extracted in step 1. It then makes a GET request to the DRS server:
+
+    GET https://drs.myexample.org/ga4gh/drs/v1/objects/314159
+
+4) The client follows any HTTP redirects returned in step 3, in case the resolver goes through an extra layer of redirection.
+
+For performance reasons, DRS clients SHOULD cache the URL pattern returned in step 2, with a suggested 24 hour cache life.
+
+==== Choosing a URI Style
+
+DRS servers can choose to issue either hostname-based or compact identifier-based DRS URIs, and can be confident that compliant DRS clients will support both.  DRS clients *must* be able to accommodate both URI types. Tradeoffs that DRS server builders, and third parties who need to cite DRS objects in datasets, workflows or elsewhere, may want to consider include:
+
+.Table Choosing a URI Style
+|===
+| |Hostname-based |Compact Identifier-based
+
+|URI Durability
+|URIs are valid for as long as the server operator maintains ownership of the published DNS address. (They can of course point that address at different physical serving infrastructure as often as they’d like.)
+|URIs are valid for as long as the server operator maintains ownership of the published compact identifier resolver namespace. (They also depend on the meta-resolvers like identifiers.org/n2t.net remaining operational, which is intended to be essentially forever.)
+
+|Client Efficiency
+|URIs require minimal client logic, and no network requests, to resolve.
+|Citation practice is consistent with publisher instructions. URIs require small client logic, and 1-2 cacheable network requests, to resolve.
+
+|Security
+|Servers have full control over their own security practices.
+|Server operators, in addition to maintaining their own security practices, should confirm they are comfortable with the resolver registry security practices, including protection against denial of service and namespace-hijacking attacks. (See the <<_appendix_compact_identifier_based_uris>> for more information on resolver registry security.)
 
-For example, if a WES server was asked to process `drs://drs.example.org/314159`, it would know that it could issue a GET request to `https://drs.example.org/ga4gh/drs/v1/objects/314159` to learn how to fetch that object.
+|https://www.nature.com/articles/sdata201618[FAIR-ness]
+|Provided the published DNS address is valid.
+|Same assuming the resolver registry namespace is valid. Slightly increased find-ability given the extra metadata available in the resolver registry (contact, organization, etc).
```

Comment:
I don't understand this row -- URI durability is definitely important for FAIRness, but that's already covered above. And the extra metadata is only at the level of the resource as a whole (vs. individual objects), which I believe is already covered by /service-info in a way that works for all URIs.

Reply:
I added because I didn't see findable, accessible, interoperable, and reproducible specifically called out in the concepts durability, security, and efficiency.  I think that's a great point about 1) service-info and 2) the service registry GA4GH is working on.  When those are both in place then I think hostname and compact identifier are equal from a FAIR perspective. That's just my opinion.

Reply:
I still think we should delete this row -- I disagree that CI-based URIs are FAIRer, and I don't want to imply that anyone who chooses to use hostname-based URIs, is being less FAIR. (Re Findability -- I think of that as being about individual objects, which is identical between the two styles.)

Reply:
Doesn't seem like this is an issue that's worth delaying 1.1 and it's not central to any functionality in the proposal so I'm just removing it.  I haven't seen any driver project care either way.